datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -0,0 +1,54 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+
+import click
+
+from ...utils import get_codeowners, get_valid_integrations
+from ..console import CONTEXT_SETTINGS, echo_failure, echo_success
+
+DIRECTORY_REGEX = re.compile(r"\/(.*)\/$")
+
+
+def get_all_checks_with_codeowners():
+    codeowners = get_codeowners()
+
+    checks_with_codeowners = set()
+    checks_with_only_entries = set()
+
+    for entry in codeowners:
+        parts = [part for part in entry.split(" ") if part != "" and part != "\t"]
+        if len(parts) < 1:
```

Comment:
what are the expected parts?
```suggestion
        if not parts:
```